test/frontend.js

```diff
@@ -396,7 +396,8 @@ describe.only('Frontend', function () {
         .goto(page('dashboard'))
         .wait(elT('forButton'))
         .click(elT('forButton'))
-        // todo check if 1st vote got through
+        .use(note('first vote clicked'))
+        .wait(elT('allVoted'))
```

Comment:
@taoeffect this is the line that solved all my problems: when we didn't wait for the first vote to go through, slow Travis lost track of things and the vote didn't go through, so the invite never got sent